src/main/java/carRacing/model/RandomGenerator.java

```diff
@@ -0,0 +1,16 @@
+package carRacing.model;
+
+import java.util.Random;
+
+public class RandomGenerator {
+
+    private static final Random random = new Random();
+
+    private static final int MAX_NUMBER = 10;
+
+    public int randomNum(){
+
+        return random.nextInt(MAX_NUMBER);
```

Comment:
```suggestion
    public int randomNum(){
        return random.nextInt(MAX_NUMBER);
```
불필요한 공백 라인은 없어도 되지 않을까?